mathesar/tests/integration/test_columns.py

```diff
@@ -22,6 +23,7 @@ def test_convert_text_column_to_number(page, go_to_patents_data_table):
     expect(page.locator(toast_box)).to_be_visible()
 
 
+@pytest.mark.skip(reason="unclear why test is failing: deferring for later")
```

Comment:
@dmos62 was this failing for you locally? Or just in CI? I just ran it locally and it passed for me. Maybe we don't need to skip it?

Reply:
It was failing for me locally. Could we try and resolve this outside this PR? I'd love to get this merged soon.

Reply:
Yep. Sounds good.